file_paths.md

```diff
@@ -0,0 +1,123 @@
+Puppet Installation Layout
+==========================
+
+This table specifies the file paths in a Puppet installation and the corresponding settings, denoted as `:setting`. The 3.x column shows the difference of the specification vs the actual 3.x implementation.
```

Comment:
Since there is only one "3x implementation", the word "actual" has no more specificity - the one and only implementation is the "actual" one :-). (I was the one that put it there, I have been reading "actual" too many times in comments - it is actually [sic] almost never needed....).